app/uk/gov/hmrc/soletraderidentificationfrontend/views/helpers/inputDate.scala.html

```diff
@@ -37,33 +39,33 @@
             )
         )
     )),
-    id = s"$id-input",
+    id = s"$id",
     items = Seq(
         InputItem(
-            id = s"$id",
-            classes = s"govuk-input--width-2${if(form(id).hasErrors || form.errors.exists(_.key.isEmpty)) " govuk-input--error" else ""}",
+            id = s"$id-day",
+            classes = s"govuk-input--width-2${DateHelper.addDayErrorClass(form)}",
             name =  s"$id-day",
             label = Some(messages("date.day")),
             value = form(s"$id-day").value,
             attributes = attributesDay
         ),
         InputItem(
             id = s"$id-month",
-            classes = s"govuk-input--width-2${if(form(id).hasErrors || form.errors.exists(_.key.isEmpty)) " govuk-input--error" else ""}",
+            classes = s"govuk-input--width-2${DateHelper.addMonthErrorClass(form)}",
             name = s"$id-month",
             label = Some(messages("date.month")),
             value = form(s"$id-month").value,
             attributes = attributesMonth
         ),
         InputItem(
             id = s"$id-year",
-            classes = s"govuk-input--width-4${if(form(id).hasErrors || form.errors.exists(_.key.isEmpty)) " govuk-input--error" else ""}",
```

Comment:
The year input is a bit small now. Can't fit 4 digits in it. Can we keep it as width-4 please

Reply:
Whoops, yes not certain how that came about.